src/TodoForm.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+function TodoForm({
+  handleSubmit,
+  onChange,
+  todoInput,
+  inputRef,
+}) {
+  return (
+    <form onSubmit={(e) => handleSubmit(e)}>
+      <input
+        name="todoInput"
+        ref={inputRef}
+        value={todoInput}
+        placeholder="할 일을 입력해 주세요"
+        onChange={(e) => onChange(e)}
```

Comment:
onChange 만 써도 충분할거 같아요!
```suggestion
        onChange={onChange}
```